BrickController2/BrickController2/BusinessLogic/PlayLogic.cs

```diff
@@ -80,7 +80,8 @@ public void ProcessGameControllerEvent(GameControllerEventArgs e)
             {
                 foreach (var controllerEvent in ActiveProfile.ControllerEvents)
                 {
-                    if (gameControllerEvent.Key.EventType == controllerEvent.EventType &&
+                    if (e.ControllerDeviceId == controllerEvent.ControllerDeviceId &&
```

Comment:
This is the reason of broken backward compatibility. This check should handle both cases:
- `controllerEvent.ControllerDeviceId` is not defined (then this match should be ignored)
-  `controllerEvent.ControllerDeviceId` has some ID then it should match it

Reply:
Yes, you're right - I fixed that:
https://github.com/vicocz/brickcontroller2/pull/75/commits/112e6b2314891e6e005851f26934a9c9a9af5d05